### PR TITLE
New /kubernetes page

### LINKED ISSUE #2163: Build containers/kubernetes
Include moving the URL and redirects

### PR DESCRIPTION
**Pull request by deadlight**
## Done
Created a new /kubernetes page

[Demo](http://www.ubuntu.com-pr-2179.run.demo.haus/kubernetes)

[Copydoc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.t6q2xj6x8p2i)

## QA
- `./run`
- In a browser, visit: [http://localhost:8001/kubernetes](http://localhost:8001/kubernetes)
- Check against the [copydoc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)
- Check the redirects like: [http://localhost:8001/containers/kubernetes](http://localhost:8001/containers/kubernetes)

Fixes #2163

### CODE REVIEW

**Comment by anthonydillon** [comment]
Please make this a full PR with just the design needed label.

**Comment by frankiegranato** [comment]
-1 :-(

A few comments for you in the copydoc.

**Comment by deadlight** [comment]
@frankiegranato I think that I got everything. Demo updated.

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2 id="support-packages">Enterprise Kubernetes support packages</h2>
+      <p>Scale-out container management is top of mind for savvy cloud architects around the world.  Ubuntu is, by far, the world&rsquo;s leading platform for container-based workflows. Canonical provides enterprise-calibre support for Kubernetes.</p>
+      <p>For more information about Kubernetes support, <a href="/containers/contact-us?product=containers-kubernetes-support" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Contact us - support packages top CTA' : undefined });">contact us&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-12">
+    <table>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Under indented.

File: templates/kubernetes/index.html [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+      <tr>
+        <th>Disaster recovery</th>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+      </tr>
+    </table>
+    <p>For more information about Kubernetes support, <a href="/containers/contact-us?product=containers-kubernetes-support" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Contact us - support packages bottom CTA' : undefined });">contact us&nbsp;&rsaquo;</a></p>
+  </div>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Missing closing `div`

File: templates/kubernetes/index.html [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+  <div class="row u-equal-height">
+    <div class="p-card col-6">
+      <div class="p-card__header">
+        <h3 class="p-heading--five">Kubernetes Explorer</h3>
+        <p class="p-heading--two">$15,000 <span class="p-heading--six">one off fee</span></p>
+      </div>
+      <ul class="p-list">
+        <p>Get up and running quickly with minimal setup, predefined architecture and on-site training.</p>
+        <p>What&rsquo;s included:</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Random `<p>`'s inside a `ul`.

File: templates/kubernetes/index.html [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+      <p><a href="https://tutorials.ubuntu.com/tutorial/install-kubernetes-with-conjure-up?_ga=2.25375934.2035833349.1504518498-2070601899.1473694436" class="p-button--brand" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Try it now', 'eventLabel' : 'Try it now - Hero CTA' : undefined });"><span class="p-link--external">Try it now</span></a>&#8195;<a href="/containers/contact-us" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact sales', 'eventLabel' : 'Contact sales - Hero CTA' : undefined });">Talk to us about Kubernetes</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--x-light is-bordered">
+  <div class="row p-divider u-equal-height">
+    <div class="col-4 p-divider__block">
+      <h3>Universal</h3>
+      <p>Consistency and compatibility across public clouds and your private data centers</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Full stops?

File: templates/kubernetes/index.html [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>Secure</h3>
+      <p>CVE updates and TLS encryption across all components</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Full stops?

File: templates/kubernetes/index.html [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>Fault Tolerant</h3>
+      <p>High availability, zero downtime upgrades, guaranteed</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Full stops?

File: templates/kubernetes/index.html [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <h2>Get started</h2>
+  </div>
+  <div class="row">
+    <div class="col-4 p-card u-align--center">
+      <p><a href="/download/server/provisioning"><img src="{{ ASSET_SERVER_URL }}adcc0253-import-icon.svg" alt="" style="max-height: 47px;" /></a></p>
+      <h4 class="u-no-margin--bottom"><a href="https://tutorials.ubuntu.com/tutorial/install-kubernetes-with-conjure-up?backURL=%2F#0"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Installation tutorial', 'eventLabel' : 'Installation tutorial - get started' : undefined });">Installation tutorial&nbsp;&rsaquo;</a></h4>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: This should have an external link.

File: templates/kubernetes/index.html [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+      <div class="p-card__header">
+        <h3 class="p-heading--five">Kubernetes Explorer</h3>
+        <p class="p-heading--two">$15,000 <span class="p-heading--six">one off fee</span></p>
+      </div>
+      <ul class="p-list">
+        <p>Get up and running quickly with minimal setup, predefined architecture and on-site training.</p>
+        <p>What&rsquo;s included:</p>
+        <li class="p-list__item is-ticked">High availability Kubernetes, deployed on cloud or VMware</li>
+        <li class="p-list__item is-ticked">Reference architecture</li>
+        <li class="p-list__item is-ticked">8 hours hands-on, web-based training</li>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Missing `NodePort and Flannel networking` list item.

File: templates/kubernetes/index.html [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+        <li class="p-list__item is-ticked">30 days of phone support</li>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Missing `*`

File: templates/kubernetes/index.html [hunk 10]
```diff
@@ -0,0 +1,406 @@
...
+      <div class="p-card__header">
+        <h3 class="p-heading--five">Kubernetes Discoverer</h3>
+        <p class="p-heading--two">$35,000 <span class="p-heading--six">one off fee</span></p>
+      </div>
+      <p>Get a customized architecture to fit your  requirements, including bare-metal environments or custom integrations, with many optional add ons.</p>
+      <p>What&rsquo;s included:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">High availability Kubernetes, deployed on Public Cloud, VMware, OpenStack, or Bare metal</li>
+        <li class="p-list__item is-ticked">Custom Kubernetes architecture optimised for your workloads</li>
+        <li class="p-list__item is-ticked">4 days on-site  in-person training</li>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Missing `Any CNI-compatible network (Calico, Flannel, Weave)` list item.

**Comment by deadlight** [comment]
@anthonydillon ready for re-review

**Pull request merged**